src/app/series/directives/series-podcast.component.ts

```diff
@@ -44,13 +56,19 @@ export class SeriesPodcastComponent implements OnDestroy, DoCheck {
       this.state = 'new';
     } else if (this.series) {
       this.state = 'creating';
+      if (!this.podcastTemplate) {
+        this.state = 'missing';
+      }
     } else {
       this.state = null;
     }
   }
 
   ngOnDestroy(): any {
     this.tabSub.unsubscribe();
+    if (this.templateSub) {
+      this.templateSub.unsubscribe();
```

Comment:
You may not actually need this.  HalDocs (underlying the `loadRelated()` call) emit a single object/array and then `finish()`, so you generally don't need to track/unsubscribe from them.  The `tabSub` is a bit different, as it's going to outlive this single tab.

That said ... this doesn't actually hurt anything.